### PR TITLE
Setup project - Basic page structure

### PR DESCRIPTION
**Pull request by elmar8287**
- Create an npm project with webpack.
- Write plain HTML markup with minimum styling
- Without API calls
- Using ES6 modules

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,186 +1,29 @@
...
-const todoWrapper = document.querySelector('.todos-wrapper');
-
-let tasks;
-/* eslint-disable */
-!localStorage.tasks ? tasks = [] : tasks = JSON.parse(localStorage.getItem('tasks')); 
-/* eslint-anable */
-
-let todoItemElems = [];
-
-function Task(description) {
```
Comment (henatan99) [inline src/index.js:12]: - [x] In this project you are required ES6 module and syntaxes. It is recommended to use the arrow function whenever necessary. Kindly refactor this and other functions to arrow functions. Kindly make the changes to all functions that can be written by the arrow function.  :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @elmar8287 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js (hunk 1 above)
Comment (elmar8287) [reply src/index.js:12]: Dear @henatan99! 🥇 

Many thanks for the review. I appreciate your comments 👍 

It seems you are looking not for the correct branch. The PR is related to "bacis-page-structure"

Many thanks and regards,

Elmar

Comment (henatan99) [reply src/index.js:12]: HI @elmar8287, my reviewer dashboard link brought me here. I don't think it is my mistake. Maybe you provided the wrong link when you submit your project. Sorry for the mishap and if this is the wrong link, reach out to student success please.

**Comment by Davidosky007** [comment]
## Hi @elmar8287 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 
thank-you-cr-gif

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/index.js (hunk 1 above)
Comment (elmar8287) [reply src/index.js:12]: Dear @henatan99,

I submit one more time and got approved. Maybe there was a wrong link. Thanks for review

**Comment by elmar8287** [comment]
Dear @Davidosky007👋

Thank you for your review! 🥇

**Pull request closed without merging**